src/test/scala/io/kx/loanapp/api/LoanAppServiceIntegrationSpec.scala

```diff
@@ -43,15 +43,17 @@ class LoanAppServiceIntegrationSpec
       val loanAppId = UUID.randomUUID.toString
       client.submit(create(loanAppId)).futureValue
       get(loanAppId, api.LoanAppStatus.STATUS_IN_REVIEW)
-      client.approve(api.ApproveCommand(loanAppId))
+
+      client.approve(api.ApproveCommand(loanAppId)).futureValue
```

Comment:
This is an async call to the service through gRPC. 

If you make two calls on the client, you don't have ordering delivery guaranteed. 
So, from time to time, the `GetCommand` can be delivered before `ApproveCommand`.